Simulation/Derivatives.m

```diff
@@ -1,9 +1,25 @@
-function StateVector_Dot = Derivatives(StateVector)
+function StateVector_Dot = Derivatives(StateVector, Object)
+  Velocity = P_dot = StateVector(3:4); % Derivative of position is velocity
+  Speed    = norm(Velocity);           % Magnitude of velocity vector
+
+  Gravity = [0, -9.81]; % Acceleration due to gravity
+
+  Density = 1.225;                              % Standard air density (kg/m3)
```

Comment:
Just a FYI, this info will probably be generated from the Atmosphere model in issue #5